db/migrate/20170703173452_add_miq_expression_enable_and_text_disable.rb

```diff
@@ -0,0 +1,6 @@
+class AddMiqExpressionEnableAndTextDisable < ActiveRecord::Migration[5.0]
+  def change
+    add_column :custom_buttons, :miq_expression_enable, :text
```

Comment:
I'm thinking `enablement_expression` or `enablement_miq_expression`.  The word enable is a verb, so it feels weird to me when it's used as a noun/adjective.